app/scripts/views/settings/devices.js

```diff
@@ -24,19 +24,20 @@ define(function (require, exports, module) {
   var FORCE_DEVICE_LIST_VIEW = 'forceDeviceList';
 
   _.extend($.timeago.settings.strings, {
-    day: t('a day'),
-    days: t('%d days'),
-    hour: t('about an hour'),
-    hours: t('%d hours'),
-    minute: t('about a minute'),
-    minutes: t('%d minutes'),
-    month: t('about a month'),
-    months: t('%d months'),
-    seconds: t('a few seconds'),
-    suffixAgo: t('ago'),
-    suffixFromNow: t('from now'),
-    year: t('about a year'),
-    years: t('%d years')
+    day: t('a day ago'),
+    days: t('a few days ago'),
+    hour: t('about an hour ago'),
+    hours: t('a few hours ago'),
+    minute: t('about a minute ago'),
+    minutes: t('a few minutes ago'),
+    month: t('about a month ago'),
+    months: t('a few months ago'),
+    seconds: t('a few seconds ago'),
+    suffixAgo: '',
+    week: t('a week ago'),
+    weeks: t('a few weeks ago'),
+    year: t('about a year ago'),
+    years: t('a few years ago')
```

Comment:
I'm looking at your branch, this string is used for > 1.5 years. Maybe this should be `more than a year ago`